src/main/java/net/earthcomputer/clientcommands/features/VillagerRngSimulator.java

```diff
@@ -36,16 +38,21 @@ public class VillagerRngSimulator {
 
     @Nullable
     private LegacyRandomSource random;
+    private long prevRandomSeed;
     private int ambientSoundTime;
+    private int prevAmbientSoundTime;
     private boolean madeSound = false;
     private int totalAmbientSounds = 0;
     private int callsAtStartOfBruteForce = 0;
     private int callsInBruteForce = 0;
     private int totalCalls = 0;
+    private int prevTotalCalls;
     private float firstPitch = Float.NaN;
     private int ticksBetweenSounds = 0;
     private float secondPitch = Float.NaN;
     @Nullable
+    private long[] seedsFromTwoPitches = null;
```

Comment:
can we rename this to `filteredSeeds` and make it so that the third pitch check is instead the nth pitch check that filters down the long[] and sets the random if there's precisely one seed left?

Reply:
I don't really see the point of making it work for more than 3 calls, most likely there's never going to be a seed where more than 3 pitches are needed since they already give much more than 48 bits of info